packages/datagateway-common/src/card/cardView.component.tsx

```diff
@@ -233,20 +259,20 @@ const CardView = (props: CardViewProps): React.ReactElement => {
   //defaultSort has been provided
   React.useEffect(() => {
     if (title.defaultSort !== undefined && sort[title.dataKey] === undefined)
-      onSort(title.dataKey, title.defaultSort, 'replace');
+      onSort(title.dataKey, title.defaultSort, 'replace', false);
```

Comment:
Minor comment, but since you specified `shiftDown?: boolean` as optional, you could leave out explicitly specifying `false` here. Would have meant less changes in the tests - but no point removing it now, just an observation